hazelcast/src/main/java/com/hazelcast/map/impl/operation/LoadAllOperation.java

```diff
@@ -61,15 +61,6 @@ public void run() throws Exception {
         recordStore.loadAllFromStore(keys);
     }
 
-    @Override
-    public void afterRun() throws Exception {
```

Comment:
This scenario is tested in `NearCacheTest` via `testAfterLoadAllWithDefinedKeysNearCacheIsInvalidated()` and `testAfterLoadAllNearCacheIsInvalidated()`.